File: src/test/testdata/inspections/staticResources/Test1.java
package main;

import com.badlogic.gdx.graphics.Texture;
import com.badlogic.gdx.maps.tiled.TiledMap;

public class Test1 {

  TiledMap map1;

  <warning>static TiledMap map2;</warning>

  void test() {
    TiledMap map3;
  }
}

File: src/test/testdata/inspections/staticResources/Test1.kt
import com.badlogic.gdx.graphics.Texture

<warning>val topLevelTexture: Texture = Texture("")</warning>

@Suppress("UNUSED_VARIABLE", "UNCHECKED_CAST", "USELESS_CAST")
public class Test {

  companion object {
    <warning>val companionTexture: Texture = Texture("")</warning>
  }

  val fieldTexture: Texture = Texture("")

  fun test() {
    val localTexture: Texture = Texture("")
  }

}
